tests/integration/src/test/resources/org/mule/test/integration/exceptions/exception-dlq-service.xml

```diff
@@ -13,8 +13,6 @@
 
     <jms:activemq-connector name="jmsConn" maxRedelivery="1" brokerURL="vm://localhost?broker.persistent=false&amp;broker.useJmx=false"/>
 
-    <jboss-transaction-manager/>
```

Comment:
i don't think you can just remove it. How do we know arjuna still works with your  change?